util/test_base.cpp

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide implementation of test_base class functions
+//
+*******************************************************************************/
+
+#include "test_base.h"
+#include "../tests/common/macros.h"
+#include "../tests/common/sycl.h"
+#include "logger.h"
+
+// conformance test suite namespace
+namespace sycl_cts {
+namespace util {
+
+void test_base::run_test(class logger &log) {
+  try {
+    this->run(log);
+  } catch (const sycl::exception &e) {
+    log_exception(log, e);
+    auto errorMsg = "a SYCL exception was caught: " + std::string(e.what());
+    FAIL(log, errorMsg);
+  } catch (const std::exception &e) {
+    auto errorMsg = "an exception was caught: " + std::string(e.what());
```

Comment:
As I suggested in another PR, what is important is that an exception was thrown.
Of course to know it we need to catch the exception, but this is an implementation detail.
```suggestion
    auto errorMsg = "a SYCL exception was thrown: " + std::string(e.what());
    FAIL(log, errorMsg);
  } catch (const std::exception &e) {
    auto errorMsg = "an exception was thrown: " + std::string(e.what());
```

Reply:
This was done in https://github.com/KhronosGroup/SYCL-CTS/pull/143 which is now merged in. So there is just some homogenizing work here.